docs/wasm/sc-dev/libs.md

```diff
@@ -51,7 +51,7 @@ Defining a Trait definition is sufficient (a contract that implements that Trait
 
 [`parity-common`](https://github.com/paritytech/parity-common) is a collection of crates that you can use in your ink! contracts.
 
-It offers all Ethereum types and is really useful if you want to port solidity code to ink!.
+It offers all Ethereum types and is really useful if you want to port solidity code to Solidity.
```

Comment:
It does not make sense - "port solidity code to Solidity"? @SUGURUSEO

Reply:
Ah you are right this shouldn't be changed. Fixed now. @andabakus